### DISCUSSION TITLE
Confused whether i was approved or rejected for benefits

### DISCUSSION THREAD

**Discussion by Dhruv2082002** [community/community#60971] [GitHub Education]
### Hi everyone,

So i applied last night twice and it direct rejected at the proof uploading the first time because of blurred document but then i uploaded a marksheet instead and that went through. i woke up and saw a rejection email and the benefits page show two requests alreadys one rejected at the bottom and one rejected at top. 

so did the first blur count as one and got rejected or the marksheet one got approved then rejected via mail?

theres no approved mail yet.

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
